main.go

```diff
@@ -129,39 +131,54 @@ func main() {
 		"enable-keep-alive",
 		defaultEnableKeepAlive,
 		"DEPRECATED: feature to enable keep alive handler in vsphere sessions. This functionality is enabled by default now")
-
 	flag.DurationVar(
 		&managerOpts.KeepAliveDuration,
 		"keep-alive-duration",
 		defaultKeepAliveDuration,
-		"idle time interval(minutes) in between send() requests in keepalive handler")
-
+		"idle time interval(minutes) in between send() requests in keepalive handler",
+	)
 	flag.StringVar(
 		&managerOpts.NetworkProvider,
 		"network-provider",
 		"",
-		"network provider to be used by Supervisor based clusters.")
+		"network provider to be used by Supervisor based clusters.",
+	)
+
+	feature.MutableGates.AddFlag(fs)
+}
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+
+	InitFlags(pflag.CommandLine)
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```

Comment:
```suggestion
        pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```
You might want to add that as well. I think it was added in CAPI a while back

Reply:
Had omitted it since I wasn't sure what exactly was the use for the normalize func, and if that is something really required. Did some digging around, seems like it replaces the `_` in flags to `-`, so say the `--enable-keep-alive` flag can also be `--enable_keep_alive` flag.
No harm having it in the code. Thanks!